R/top-level_scraper.R

```diff
@@ -455,27 +455,33 @@ fast_scraper <- function(game_ids,
 # roster ------------------------------------------------------------------
 
 #' Load Team Rosters for Multiple Seasons
-#' @inherit nflreadr::load_rosters
+#' @details See [`nflreadr::load_rosters`] for details.
 #' @seealso For information on parallel processing and progress updates please
 #' see [nflfastR].
 #' @examples
 #' \donttest{
 #' # Roster of the 2019 and 2020 seasons
 #' fast_scraper_roster(2019:2020)
 #' }
+#' @name fast_scraper_roster
+NULL
+
 #' @export
 fast_scraper_roster <- nflreadr::load_rosters
 
 # schedules ---------------------------------------------------------------
 
 #' Load NFL Season Schedules
-#' @inherit nflreadr::load_schedules
+#' @details See [`nflreadr::load_schedules`] for details.
 #' @seealso For information on parallel processing and progress updates please
 #' see [nflfastR].
-#' @export
 #' @examples
 #'\donttest{
 #' # Get schedules for the whole 2015 - 2018 seasons
 #' fast_scraper_schedules(2015:2018)
 #' }
+#' @name fast_scraper_schedules
+NULL
+
+#' @export
 fast_scraper_schedules <- nflreadr::load_schedules
```

Comment:
This method actually introduces a weird build time dependency problem. I think this should be a function that passes all arguments to the appropriate nflreadr via … (and then uses inheritDotParams. Otherwise, there’s a mismatch when calling nflfastR version of function